airport-challenge-solution.md

```diff
@@ -0,0 +1,158 @@
+## Catherine's airport challenge solution
+
+## Part 1 User Story
+
+As an air traffic controller
+So I can get passengers to a destination
+I want to instruct the airport to land a plane
+
+--think of airport as basket, and plane as (bagel)item from Bob's Bagels exercise--
+
+## Part 1 Domain Model
+
+| Objects | Properties                  | Messages          | Outputs |
+| ------- | --------------------------- | ----------------- | ------- |
+| Airport | landedPlanes @Array[@Plane] | landPlane(@Plane) | @Void   |
+|         |                             |                   |         |
+| Plane   | id @String                  | getId()           | @String |
+
+## Tests 1-3
+
+a Need an Airport
+b Need to be able to give the Airport a Plane
+c LandedPlanes list length should increase by 1 when a Plane is added to the empty LandedPlanes list
+
+**Test 1** - `landedPlanes` length increases by 1 when `landPlane` is called with _instance of `Plane`_
+
+**Test 2** - `landPlane` should only add `Plane` instances to the `landedPlanes`
+
+**Test 3** - edge Case - falsy values should not be added to the Airport
+
+//I am not clear if if Test 2 and 3 are the same explicitly?
```

Comment:
I understand that test 2 lands a plane, but test 3 needs to exist to check if adding something else is possible, for which I tried a string and an integer.